README.md

```diff
@@ -1,2 +1,18 @@
 # devops
-Tools, build pipelines etc for Azure DEVOPS
+Tools, build pipelines etc for Azure DEVOPS. Project follows [gitflow]() for development&releases.
+
+## Scripts
+
+This section contains links to documentation about available scripts
+
+### [init-repo](docs/init-repo.md)
+
+Script, that will initialize your repository with develop branch and pull requests policies over master, develop, feature etc. branches.
+
+## How to contribute
+
+See [contributing.md](CONTRIBUTING.md).
+
+### Signed commits
+
+You need to have signed commits
```

Comment:
please add some link to documentation how to achieve this

Reply:
sure, thanks. Will do.

Reply:
done.